text_localization_environment/TextLocEnv.py

```diff
@@ -86,17 +86,14 @@ def calculate_reward(self, action):
             else:
                 reward = -self.ETA
         else:
-            new_iou = self.compute_best_iou()
-            reward = np.sign(new_iou - self.iou)
-
-            if reward == 0:
-                self.steps_since_last_change += 1
-            else:
-                self.steps_since_last_change = 0
-
-            if self.steps_since_last_change >= 3:
-                reward = -1
```

Comment:
@FMalerba Are you sure that lines 92–98 should have been removed? Or did you maybe accidentally delete those lines during the merge conflict resolving?

Reply:
Yeah I don't know how to revert these changes.... Sourcetree gives me an error if I try to revert this commit